content/docs/command-reference/remote/modify.md

```diff
@@ -406,7 +406,42 @@ application. To use a custom authentication method, use the following parameters
   ```dvc
   $ dvc remote modify myremote allow_anonymous_login true
   ```
+**Authentication examples:**
 
+Authentication example with `connection_string`:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify --local myremote connection_string 'mysecret'
+$ dvc remote push
+```
+  
+Authentication example with currently logged in az cli user/identity:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify myremote account_name 'myaccount'
+$ dvc remote push
+```
+  
+Authentication example with `account_key`:
```

Comment:
No example for `tenant_id` + `client_id` / `client_secret`?

Reply:
I could not test these, since we're not using service principals for authentication.

Reply:
let's clarify in the title that this list is not comprehensive, that we have here _some_ possible ways to auth